apps/webcomponents/src/app/components/gn-search-input/gn-search-input.component.html

```diff
@@ -1,3 +1,8 @@
 <div id="angular-app-root">
-  <gn-ui-fuzzy-search class="text-[18px]"> </gn-ui-fuzzy-search>
+  <gn-ui-fuzzy-search
+    class="text-[18px]"
+    (inputSubmited)="search($event)"
```

Comment:
Hmm, the issue is that if `datahub-url` is not defined, these handlers will still be set which means that the fuzzy search input will not update the search state, so it becomes useless.

See https://geonetwork.github.io/geonetwork-ui/wc-search/demo/webcomponents/gn-search-input-and-results.sample.html